npc/re/quests/quests_16_1.txt

```diff
@@ -13205,6 +13436,35 @@ prt_cas,98,248,5	script	Cat#ep16bgm	4_CAT,{
 	close;
 }
 
+prt_cas,90,121,5	script	Hiba Ajif#ep16	4_M_REDSWORD,{
```

Comment:
```suggestion
prt_cas,98,231,5	script	Hiba Ajif#ep16	4_M_REDSWORD,{
```

Reply:
The npcs exist but they are commented in our version. Maybe they changed them with the server/version. Your reference is kro?

Reply:
Yep, kro sakray